README.md

```diff
@@ -239,14 +304,33 @@ This image is to be used in the run step.
 
 ## Run
 
-Run the resulting image with the `run-...` scripts.
+Run the resulting image with the `qemu-system-*` commands.
+In order to run `nginx` you need to first set up a network bridge.
+You can do that by running the following commands:
+
+```console
```

Comment:
This is too complicated. I say we re-add the `run-...` scripts. This is what we have for [`app-nginx`](https://github.com/unikraft/app-nginx), and I would keep it here. Makes it easier to test.